app/models/form526_submission.rb

```diff
@@ -52,36 +52,40 @@ def start
       workflow_batch = Sidekiq::Batch.new
       workflow_batch.on(
         :success,
-        'Form526Submission#start_evss_submission',
+        'Form526Submission#rrd_complete_handler',
         'submission_id' => id
       )
       job_ids = workflow_batch.jobs do
         rrd_processor_class.perform_async(id)
       end
       job_ids.first
     else
-      start_evss_submission(nil, { 'submission_id' => id })
+      start_evss_submission_job
     end
   rescue => e
     Rails.logger.error 'The fast track was skipped due to the following error ' \
-                       " and start_evss_submission wass called: #{e}"
-    start_evss_submission(nil, { 'submission_id' => id })
+                       " and start_evss_submission_job is being called: #{e}"
+    start_evss_submission_job
   end
 
   def rrd_process_selector
     @rrd_process_selector ||= RapidReadyForDecision::ProcessorSelector.new(self)
   end
 
+  # Afer RapidReadyForDecision is complete, this method is
+  # called by Sidekiq::Batch as part of the Form 526 submission workflow
+  def rrd_complete_handler(_status, options)
+    submission = Form526Submission.find(options['submission_id'])
+    submission.start_evss_submission_job
+  end
+
   # Kicks off a 526 submit workflow batch. The first step in a submission workflow is to submit
   # an increase only or all claims form. Once the first job succeeds the batch will callback and run
   # one (cleanup job) or more ancillary jobs such as uploading supporting evidence or submitting ancillary forms.
   #
   # @return [String] the job id of the first job in the batch, i.e the 526 submit job
   #
-
-  def start_evss_submission(_status, options)
-    submission = Form526Submission.find(options['submission_id'])
-    id = submission.id
+  def start_evss_submission_job
```

Comment:
Note the `_job` method suffix for all methods that use `Sidekiq::Batch` to start a new Sidekiq job.